Docker.md

```diff
@@ -0,0 +1,43 @@
+# V-lang development container in docker 
+
+This document describes how to set up a development environment for developing V-language inside a docker container. The docker container will provide most dependencies you need.
+
+## The container 
+
+This image contains:
+- **Debian Buster**, latest version (will be moved to alpine when supported)
+- **gcc** and **clang** compilers
+- **nodejs** and **npm**
+- **git**
+- **open-ssh client** to support github push over ssh
+- **Valgrind**
```

Comment:
Use markdown instead of bold case

Reply:
Well not sure what you mean? That is valid markdown right?